### ISSUE TITLE
Hours REVERT to normal after idling with singleboostr

### ISSUE THREAD

**Issue by stolond** [Ezzpify/HourBoostr#154]
After i idle for like 10 hours with singleboostr and i click Stop Idle, game syncs and hours revert to how they were before. Any solutions?

**Comment by rishav394** [comment]
Yes you need to wait some time to let the changes have an effect. Try hopping into a game manually and then close it once again to make sure. Also make sure that your internet is not interrupted while boosting.
